test/test_binary_ufuncs.py

```diff
@@ -1425,31 +1425,34 @@ def test_fmod(self, device, dtype):
         def _reference_implementation(x, mod):
```

Comment:
This name seems odd. A reference implementation would typically return the same value as expected by the actual operation. This, however, seems to perform the test. Maybe a name like "_helper" would be more appropriate?

Reply:
It makes sense! Will change it.